docs/blog/2020-02-17-changing-the-oss-status-quo/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Changing the Open Source Status Quo"
+date: 2020-02-17
+author: Aisha Blake
+excerpt: "How can we increase the diversity of open source contributors? As maintainers, we need to listen to a diverse group of community members and make thoughtful changes to create a truly inclusive project."
+tags:
+  - open-source
+  - community
+---
+
+In business and in our individual careers, so many of us rely heavily on open source software. We need it but open source isn't as open as it's often made out to be. Too many projects are unfriendly, even openly hostile, to new contributors. This effect, combined with the intersection of many underrepresented identities in tech effectively shuts people out of the process. Black, indigenous, and people of color (BIPOC) and women are at some of the most visible of these intersections. There are many more. Consider _who_ contributes to open source.
```

Comment:
```suggestion
In business and in our individual careers, so many of us rely heavily on open source software. We need it but open source isn't as open as it's often made out to be. Too many projects are unfriendly, even openly hostile, to new contributors. This effect, combined with the intersection of many underrepresented identities in tech, effectively shuts people out of the process. Black, indigenous, and people of color (BIPOC) and women are at some of the most visible of these intersections. There are many more. Consider _who_ contributes to open source.
```
Adding a comma here, let me know if you think it's misplaced. One other addition you could consider is to conclude the paragraph with a teaser of what the post will be about, i.e. "Consider _who_ contributes to open source; that's what we'll do in this post."